bedrock/mozorg/views.py

```diff
@@ -422,7 +422,7 @@ def new_home(request, locale=None):
 
 def home(request):
```

Comment:
Now that we only have one home page template, do we need these two different views? Do we still need the `HomeTestView` class above? It seems all code paths now lead to the same `home.html` template.